### PR TITLE
chore(IT Wallet): [SIW-1440] Add Disability Card credential issuance

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the Disability Card details in the credential details screen, such as the QR Code and the ability to display boolean claims.
It also contains small improvements and unused code removals, as listed in the changes below.

## List of changes proposed in this pull request
- Fixed `useItwDisableGestureNavigation` hook name
- Added static logo and name for the credential's issuer in the `ItwIssuanceCredentialTrustIssuerScreen` component
- Added `ItwGenericErrorComponent` which displays a generic failure and used it instead of `costNull`
- Added `BoolClaim` decoder and `BoolClaimItem` component, which renders a "Yes/No" claim in the credential preview/details screen
- Added `UrlClaim` decoder to decode URL type claims
- Added `QrCodeImage` component to display a QR Code image from a string value
- Added `ItwQrCodeClaimImage` component, which wraps `QrCodeImage` for the ITW QR Code claims
- Added `WellKnownClaim` enum, which describes the claims that we need to manipulate manually in order to render the correct information
- Added `getHumanReadableParsedCredential`, which helps to print data in a human readable format
- Added credential issuance buttons to ITW playgrounds
- Changed `ignoreMissingAttributes` parameter to `false` in the `verifyAndParseCredential` function call during the issuance of a credential: this will let us know if the credential claims does not conform to the received EC
- Refactored `ItwCredentialsClaimsSection` component: 
  - it has been moved to the `presentation` folder and it now uses `ItwCredentialClaimsList` component
  - it can display a QR Code if the credential has one, using the new `ItwQrCodeClaimImage` component 
- Moved `ItwCredentialClaimsSection` from `common` to `presentation` folder
- Removed everything related to claims section, which are no more needed: `ClaimSection`, `sectionsByClaim`, `groupCredentialClaims`
- Removed `itwErrorUtils.ts` utilities
- Removed `useItwIdpIdentification.ts` hook

## How to test
In the PRE env, start the Disability Card issuance from the IT Wallet Playground. You should be able to obtain the credential and see it in your wallet.

### CODE REVIEW

File: ts/features/itwallet/common/utils/itwClaimsUtils.ts [hunk 1]
```diff
@@ -27,29 +25,24 @@ import { ParsedCredential, StoredCredential } from "./itwTypesUtils";
...
+   * Unique ID must be excluded from every credential and should not rendered in the claims list
+   */
+  unique_id = "unique_ID",
+  /**
+   * Claim used to extract expiry date from a credential. This is used to display how many days are left for
+   * the credential expiration or to know if the credential is expired
+   */
+  expiry_date = "expiry_date",
+  /**
+   * Claim used to display a QR Code for the Disability Card. It must be exlcuded from the common claims list
```
Comment (gispada) [inline ts/features/itwallet/common/utils/itwClaimsUtils.ts:41]: Small typo: `exlcuded`!

File: ts/features/itwallet/common/utils/itwClaimsUtils.ts [hunk 2]
```diff
@@ -27,29 +25,24 @@ import { ParsedCredential, StoredCredential } from "./itwTypesUtils";
  */
 
 /**
- * Retrieves the organization name from the evidence claim of the given credential.
- * If the evidence claim is not present or cannot be decoded, a fallback value is returned.
- * @param credential - The parsed credential object.
- * @returns The organization name from the evidence claim or a fallback value.
+ * We strongly discurage direct claim manipulation, but some spacial cased must be addressed with direct access
```
Comment (gispada) [inline ts/features/itwallet/common/utils/itwClaimsUtils.ts:28]: Small typo: `discurage`!

File: ts/features/itwallet/presentation/components/ItwPresentationClaimsSection.tsx [hunk 3]
```diff
@@ -37,14 +38,11 @@ export const ItwCredentialClaimsSection = ({
...
-            {index !== 0 && <Divider />}
-            <ItwCredentialClaim claim={claim} hidden={valuesHidden} />
-          </React.Fragment>
-        ))}
-      </View>
+      <ItwQrCodeClaimImage
+        claim={data.parsedCredential.link_qr_code}
+        valuesHidden={valuesHidden}
+      />
+      <ItwCredentialClaimsList data={data} />
```
Comment (gispada) [inline ts/features/itwallet/presentation/components/ItwPresentationClaimsSection.tsx:45]: Did you forget to pass the `isHidden` prop? It's no longer possible to hide claims.

File: ts/features/itwallet/common/utils/itwClaimsUtils.ts (hunk 1 above)
Comment (mastro993) [reply ts/features/itwallet/common/utils/itwClaimsUtils.ts]: Addressed in https://github.com/pagopa/io-app/pull/6074/commits/b07750f59059602857b02be7907fadd88ca2c35f

File: ts/features/itwallet/common/utils/itwClaimsUtils.ts (hunk 2 above)
Comment (mastro993) [reply ts/features/itwallet/common/utils/itwClaimsUtils.ts]: Addressed in https://github.com/pagopa/io-app/pull/6074/commits/b07750f59059602857b02be7907fadd88ca2c35f and https://github.com/pagopa/io-app/pull/6074/commits/f4386520338ea40306c1f9363ad4268a4fbf508f

File: ts/features/itwallet/presentation/components/ItwPresentationClaimsSection.tsx (hunk 3 above)
Comment (mastro993) [reply ts/features/itwallet/presentation/components/ItwPresentationClaimsSection.tsx]: Nice catch! Thank you
Addressed here https://github.com/pagopa/io-app/pull/6074/commits/6bc44033749b9f19684ad3e8882b7f315230066c

File: locales/it/index.yml [hunk 4]
```diff
@@ -3332,6 +3332,9 @@ features:
         personalDataTitle: "Dati personali"
         documentDataTitle: "Dati documento"
         lastUpdated: "Dati aggiornati al {{lastUpdateTime}}"
+        boolClaim:
+          true: Si
```
Comment (LazyAfternoons) [inline locales/it/index.yml:3336]:
```suggestion
          true: Sì
```

File: locales/en/index.yml [hunk 5]
```diff
@@ -3332,6 +3332,9 @@ features:
         personalDataTitle: "Personal data"
         documentDataTitle: "Document data"
         lastUpdated: "Last updated on {{lastUpdateTime}}"
+        boolClaim:
+          true: Si
```
Comment (LazyAfternoons) [inline locales/en/index.yml:3336]:
```suggestion
          true: Sì
```

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]

**Review by LazyAfternoons — APPROVED** [review]
LGTM

**Pull request merged**